packages/flutter_tools/gradle/src/main/groovy/flutter.groovy

```diff
@@ -256,30 +257,7 @@ class FlutterPlugin implements Plugin<Project> {
             }
         }
 
-        // Load shared gradle functions
-        project.apply from: Paths.get(flutterRoot.absolutePath, "packages", "flutter_tools", "gradle", "src", "main", "groovy", "native_plugin_loader.groovy")
-
-        FlutterExtension extension = project.extensions.create("flutter", FlutterExtension)
-        Properties localProperties = new Properties()
-        File localPropertiesFile = rootProject.file("local.properties")
-        if (localPropertiesFile.exists()) {
-            localPropertiesFile.withReader("UTF-8") { reader ->
-                localProperties.load(reader)
-            }
-        }
-
-        Object flutterVersionCode = localProperties.getProperty("flutter.versionCode")
-        if (flutterVersionCode == null) {
-            flutterVersionCode = "1"
-        }
-        extension.flutterVersionCode = flutterVersionCode
-
-        Object flutterVersionName = localProperties.getProperty("flutter.versionName")
-        if (flutterVersionName == null) {
-            flutterVersionName = "1.0"
-        }
-        extension.flutterVersionName = flutterVersionName
-
```

Comment:
@camsim99 this code was added in #141417

Reply:
Ok adding this back